ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -32,6 +32,7 @@ import {ReactFragment, useState} from 'react';
 import { Link, RouteComponentProps, withRouter } from 'react-router-dom';
 import {isUsingFreeTierBillingAccount} from 'app/utils/workspace-utils';
 import { EgressEventsTable } from './egress-events-table';
+import {AdminLockRequest} from './admin-lock-request';
```

Comment:
nit: relative path in import. can we use the absolute path `app/pages/admin/admin-lock-request`?

Reply:
I dunno.  This is a very common pattern, especially when it's a related component like it is here.